olwebsite-app/src/style-sheets/pages/_partners-view.scss

```diff
@@ -0,0 +1,7 @@
+.partners-view-container {
+    width: 100%;
```

Comment:
the view containers seem to have the same exact rules, would it be useful to make a generic "view" sass page that can make sure all the css is aligned